doc/content/concepts/email-templates.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Email Templates"
+description: ""
+weight: 15
+---
+
+## What is it?
+
+Email templates define the contents of the emails that The Things Stack sends to its users. They allow users to override the default contents of the emails with their own custom contents.
```

Comment:
> They allow users to override

I think "users" should be "network operators" here.

@johanstokking is "network operators" the term we now use for the people that deploy the stack?

Reply:
Yeah good catch. In this paragraph the mentioned "users" mean two different things. I think "network operators" is a better term here indeed.